BUILD

```diff
@@ -59,13 +59,13 @@ deploy_brew(
     version_file = "//:VERSION"
 )
 
-linux_packages(
+deb_package(
+    name = "deploy-deb",
     package_name = "grakn-core-bin",
-    maintainer = "Max Vorobev <vmax0770@gmail.com>",
+    maintainer = "Grakn Labs <support@grakn.ai>",
     description = "Grakn Core (binaries)",
     version_file = "//:VERSION",
     installation_dir = "/opt/grakn/core/",
-    rpm_spec_file = "//dependencies/distribution/rpm:grakn-core-bin.spec",
     empty_dirs = [
         "./var/log/grakn/",
```

Comment:
let's be consistent with the relative path by removing the leading `./`: `var/log/grakn/"`

Reply:
@lolski fixed in https://github.com/graknlabs/grakn/pull/4777/commits/1456af49e17ee353ec93c7b05a844d2ff421ab7c